README.md

```diff
@@ -8,7 +8,7 @@ For now the crate can be included by branch or by tag via git.
 
 ```toml
 #[dependencies]
-unicode-categories = { git = "https://github.com/ncatelli/unicode_categories", branch = "main" }
+unicode-categories = { git = "https://github.com/ncatelli/unicode-categories.git", branch = "main" }
```

Comment:
This should work essentially equivalently to how it was before. Was this failing for you?

Reply:
Yep

Reply:
Oh i see `_` vs `-`. Good catch. Thank you.

Reply:
Oh I didn't see the `.git`, sorry for the confusion 😅